src/modules/filesystem/filesystem.c

```diff
@@ -307,5 +307,6 @@ size_t lovrFilesystemWrite(const char* path, const char* content, size_t size, b
   }
 
   size_t bytesWritten = lovrFileWrite(&file, (void*) content, size);
-  return lovrFileDestroy(&file), bytesWritten;
+  lovrFileDestroy(&file);
+  return bytesWritten;
```

Comment:
This isn't a mistake, it's using the comma operator to cause a side effect and then return a value.  I kinda like using this idiom in certain situations.